assets/src/scss/base/_variables.scss

```diff
@@ -6,6 +6,9 @@ $large-width:             992px;
 $extra-large-width:       1200px;
 $extra-extra-large-width: 1600px;
 
+// Override html selectors
+$checkbox-size: 20px;
```

Comment:
Using this value as variable since it's being used in both files. It will avoid to hve conflict with sizes.